frontend/chat-plugin/src/components/chat/index.tsx

```diff
@@ -40,8 +40,13 @@ const Chat = (props: Props) => {
   const [isChatHidden, setIsChatHidden] = useState(true);
   const [messages, setMessages] = useState([welcomeMessage]);
 
+  const getResumeToken = () => {
+    const queryParams = new URLSearchParams(window.location.search);
+    return queryParams.has('resume_token') ? queryParams.get('resume_token') : null;
```

Comment:
Hm, doesn't it automatically return null if the key does not exist?

Reply:
```suggestion
    return queryParams.get('resume_token');
```

should do it (as @bitboxer is also saying :))